mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Permission to determine as precise a location as possible from the available location providers, including the GPS, WiFi and mobile cell data. cc @BenjaminSsempala

Reply:
Thanks @NoahNsimbe This is great